### PR TITLE
Showing options menu and navigation drawer using javascript & also getting nextTime1-4

### PR DESCRIPTION
**Pull request by infinyte7**
## Purpose / Description
Get nextTime1, nextTime2, nextTime3, nextTime4 in WebView & showing options menu and navigation drawer by javascript function called from WebView 
This is continuation #6307

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?
Tested on android emulator
For showing Options Menu and Navigation Drawer using JavaScript
```
<button onclick="ankiShowOptionsMenu()">Show Options Menu</button>
<button onclick="ankiShowNavDrawer()">Show Navigation Drawer</button>
```

For getting time info
```
<script>
   console.log(AnkiDroidJS.ankiGetNextTime1());
   console.log(AnkiDroidJS.ankiGetNextTime2());
   console.log(AnkiDroidJS.ankiGetNextTime3());
   console.log(AnkiDroidJS.ankiGetNextTime4());
</script>
```

## Learning (optional, can help others)
https://developer.android.com/reference/android/webkit/WebView

## Checklist
_Please, go through these checks before submitting the PR._

- [ x ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ x ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ x ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ x ] You have commented your code, particularly in hard-to-understand areas
- [ x ] You have performed a self-review of your own code

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java [hunk 1]
```diff
@@ -462,7 +464,7 @@ public boolean onTouch(View v, MotionEvent event) {
     };
 
     @SuppressLint("CheckResult")
-    //This is intentionally package-private as it removes the need for synthetic accessors
+        //This is intentionally package-private as it removes the need for synthetic accessors
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:467]: unrelated whitespace, please remove

Comment (infinyte7) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:467]: Okay, I will keep it in mind. I will experiment and learn GitHub workflow then create clean PR. Thanks

Comment (infinyte7) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:467]: The idea for #6328 is check for AnkiDroid Version if It is greater than 2.11 then ```init()``` other wise do nothing or something?
Also add that which function is available in which version of AnkiDroid.
I just need some more explanation for its working.

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:467]: I added clarifying comments over there to keep them in one spot

File: AnkiDroid/src/main/assets/scripts/card.js [hunk 2]
```diff
@@ -194,4 +204,4 @@ var onPageFinished = function() {
             });
         }
     }
-}
+}
```
Comment (mikehardy) [inline AnkiDroid/src/main/assets/scripts/card.js:207]: Still need to make sure whitespace is not modified unnecessarily

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java (hunk 1 above)
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:467]: Still need to make sure whitespace is not modified unnecessarily

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java [hunk 3]
```diff
@@ -3399,5 +3411,25 @@ public boolean ankiGetCardMark() {
...
+            return (String) mNext3.getText();
+        }
+
+        @JavascriptInterface
+        public String ankiGetNextTime4() {
+            return (String) mNext4.getText();
+        }
     }
-}
+}
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:3435]: Still need to make sure whitespace is not modified unnecessarily

**Comment by david-allison-1** [comment]
@infinyte7 This branch can't be merged due to conflicts - looks like you'll need to rebase your commit(s) onto our current master and force push this branch before it can be merged.

**Comment by infinyte7** [comment]
I will create pull request after cleaning my github repo.

**Comment by mikehardy** [comment]
Okay - and just to be clear, I'm still positive on the change, just that as you are contributing regularly it's a useful time to get the PR style nailed down. Thanks @infinyte7  :-)
